cmake/ActsCompilerOptions.cmake

```diff
@@ -5,14 +5,14 @@ if (NOT CMAKE_BUILD_TYPE)
 endif() 
 
 set(ACTS_CXX_FLAGS "-Wall -Wextra -Wpedantic -Wshadow -Wunused-local-typedefs")
-set(ACTS_CXX_FLAGS_DEBUG "--coverage")
+#set(ACTS_CXX_FLAGS_DEBUG "--coverage")
 set(ACTS_CXX_FLAGS_MINSIZEREL "")
 set(ACTS_CXX_FLAGS_RELEASE "")
 set(ACTS_CXX_FLAGS_RELWITHDEBINFO "")
 
 # Acts linker flags
-set(ACTS_EXE_LINKER_FLAGS_DEBUG "--coverage")
-set(ACTS_SHARED_LINKER_FLAGS_DEBUG "--coverage ")
+#set(ACTS_EXE_LINKER_FLAGS_DEBUG "--coverage")
+#set(ACTS_SHARED_LINKER_FLAGS_DEBUG "--coverage ")
```

Comment:
Ouch... These should not be committed. We had to remove these flags to make DPC\+\+ work in Debug mode, but we will need to sort this out in a nicer way in the CMake configuration later on. Not by torpedoing the host code's coverage tests.

Reply:
Please undo your change in line 8 as well. I meant all of them, not just these 2 lines.